src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -230,12 +230,12 @@ object BreakpointCategory extends Enumeration {
  * @param id                    Combined ID retaining the IDs of all constituent breakpoints
  * @param category              Breakpoint category
  * @param left_contig           Contig name for left side of breakpoint
- * @param left_min_pos          Minimum coordinate of left breakends
- * @param left_max_pos          Maximum coordinate of left breakends
+ * @param left_min_pos          Minimum coordinate of left breakends (1-based)
+ * @param left_max_pos          Maximum coordinate of left breakends (1-based)
  * @param left_strand           Strand at left breakends
  * @param right_contig          Contig name for right side of breakpoint
- * @param right_min_pos         Minimum coordinate of right breakends
- * @param right_max_pos         Maximum coordinate of right breakends
+ * @param right_min_pos         Minimum coordinate of right breakends (1-based)
+ * @param right_max_pos         Maximum coordinate of right breakends (1-based)
```

Comment:
do we want to say inclusive here, or is that implied?

Reply:
Can't hurt, I'll add it!